solutions/problem-006/README.md

```diff
@@ -0,0 +1,12 @@
+```
+n = 100
+t1 = 0.25*(n**4 + n**2 + 2*(n**3))
+t2 = (n*(n+1)*(2*n+1))/6
+
+print(int(t1-t2))
+```
```

Comment:
add a bit of code documentation as well.
try to explain it in markdown also, the problem statement

Reply:
Can you check now, please?